app/src/main/java/org/systers/mentorship/view/activities/SettingsActivity.kt

```diff
@@ -20,9 +26,27 @@ class SettingsActivity : BaseActivity() {
         supportActionBar?.setDisplayShowHomeEnabled(true);
 
         tvLogout.setOnClickListener {
-            preferenceManager.clear()
-            startActivity(Intent(this, LoginActivity::class.java))
-            finishAffinity()
+
+            val builder = AlertDialog.Builder(this)
+            builder.setTitle("CONFIRM LOGOUT")
+            builder.setMessage("Are you sure you want to logout ?")
+
+            builder.setPositiveButton("YES"){dialog, which ->
+
+                preferenceManager.clear()
+                startActivity(Intent(this, LoginActivity::class.java))
+                finishAffinity()
+
+            }
+
+            builder.setNegativeButton("No"){dialog,which ->
+
```

Comment:
I think it's better to add **dialog.cancel()** here instead of leaving blank.

Reply:
1. "No" is inconsistent with the other capitalized text you have added. Please be consistent.
2. The text for this button should also be in the strings.xml file.